docs/example_qiskit_entangle.py

```diff
@@ -21,13 +22,19 @@
 
 from quantuminspire.qiskit import QI
 
+QI_EMAIL = os.getenv('API_USERNAME')
+QI_PASSWORD = os.getenv('API_PASSWORD')
+
 
 def get_authentication():
     """ Gets the authentication for connecting to the Quantum Inspire API."""
-    print('Enter email:')
-    email = input()
-    print('Enter password')
-    password = getpass()
+    if QI_EMAIL and QI_PASSWORD:
```

Comment:
This is a bit implicit, we should be more explicit here I think
```suggestion
    if QI_EMAIL is not None and QI_PASSWORD is not None:
```

Reply:
oke